src/components/layouts/AppLayout.vue

```diff
@@ -1,10 +1,242 @@
+          style="
+            clip-path: polygon(
+              74.1% 44.1%,
+              100% 61.6%,
+              97.5% 26.9%,
+              85.5% 0.1%,
+              80.7% 2%,
+              72.5% 32.5%,
+              60.2% 62.4%,
+              52.4% 68.1%,
+              47.5% 58.3%,
+              45.2% 34.5%,
+              27.5% 76.7%,
+              0.1% 64.9%,
+              17.9% 100%,
+              27.6% 76.8%,
+              76.1% 97.7%,
+              74.1% 44.1%
+            );
+          "
+        />
+      </div>
+      <div class="grid lg:grid-cols-2">
+        <div class="mx-auto max-w-2xl py-32 sm:py-48 lg:py-56">
+          <div class="text-center">
+            <h1
+              class="text-4xl font-bold tracking-tight text-gray-900 sm:text-6xl"
+            >
+              Open Source UI Library for Vue3
+            </h1>
+            <div class="hidden sm:mb-8 sm:flex sm:justify-center mt-8">
+              <div
+                class="relative rounded-full px-3 py-1 text-sm leading-6 text-gray-600 ring-1 ring-gray-900/10 hover:ring-gray-900/20"
+              >
+                Crafted by HitPay
+                <a
+                  href="https://hitpayapp.com"
+                  class="font-semibold text-indigo-600"
+                  ><span class="absolute inset-0" aria-hidden="true" />Read more
+                  <span aria-hidden="true">&rarr;</span></a
+                >
+              </div>
+            </div>
+            <p class="mt-6 text-lg leading-8 text-gray-600">
+              The Open Source UI Library for Vue3 offers a wide range of
+              components and tools to streamline the development process for
+              Vue.js applications. With a focus on flexibility and
+              customization, this library provides developers with the ability
+              to create modern and responsive user interfaces with ease.
+              <br />
+              <br />
+              Whether you are building a simple website or a complex web
+              application, this UI library has everything you need to enhance
+              the user experience and improve overall functionality.
+            </p>
+            <div class="mt-10 flex items-center justify-center gap-x-6">
+              <a
+                href="https://github.com/hit-pay/orchid"
+                class="rounded-md bg-slate-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:bg-slate-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-slate-600"
+                >GitHub ⭐
+              </a>
+              <a
+                href="/storybook/"
+                class="rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
+                >StoryBook ⭐
+              </a>
```

Comment:
Also woth this link meybe we could creare array something like:
```
[
{
title: 'Title',
link: 'http//....',
}
]
```
And run it in v-for?